src/crime_vancouver_eda.md

```diff
@@ -0,0 +1,55 @@
+# Exploratory data analysis of Crimes in Vancouver data set
```

Comment:
I guess we also have to make it as Rmd such that you can link the bibliography and citations.
Can you add an issue so that this is in-sync with Jasmine together?

Reply:
I checked sample project didn't see the bib file. I think they did in-text markdown citation.
But sure, I can make a issue for this

Reply:
The sample has this too, see the bottom section, Reference.
https://github.com/ttimbers/breast_cancer_predictor/blob/master/src/breast_cancer_eda.md

Please refer to the Rmd version and then you should be able to add this, can reference to the .bib Jasmine just created, and we can refer to the same bib.

Reply:
Upload Rmd with reference